_episodes/05-accelerating-lammps.md

```diff
@@ -477,19 +496,20 @@ each GPU. The detail about the graphics card is also printed, along
 with the *numerical precision* used by the **GPU** package is also printed. In this case, it
```

Comment:
@prithwish-ichec, same as above, `{{ site.sched_comment }} {{ site.sched_flag_ntasks }}`, `SBATCH` and `ntasks`. Please confirm

Reply:
I didn't get this. Just to say: number of MPI tasks per GPU * number of GPUs used in a calculation (i.e. --gres=gpu:2) = ntasks-per-node (i.e. 24 in this case)